E3Next/Settings/CharacterSettings.cs

```diff
@@ -538,6 +540,7 @@ public IniData CreateSettings(string fileName)
                 section.Keys.AddKey("Who to HoT", "");
                 section.Keys.AddKey("Pet Owner", "");
                 section.Keys.AddKey("Auto Cast Necro Heal Orbs (On/Off)", "On");
+                section.Keys.AddKey("Resurrection", "");
```

Comment:
Default this to the list you pulled out of the `Rez` class

Reply:
Working on this now. Seems to be a bug with merging sections of ini files with multiple values that will need to be fixed to render out the multiple options.